### ISSUE TITLE
[Connection issue] with PyMongo

### ISSUE THREAD

**Issue by greed2411** [Introduction-to-Tornado/Introduction-to-Tornado#18]
Hey,

I was going through the book, and am at the databases chapter, according to the code provided in the book.

```
>>> import pymongo
>>> conn = pymongo.Connection("localhost", 27017)
```

it throws the error,
`AttributeError: module 'pymongo' has no attribute 'Connection'`

googled the issue and found it out and fixed it,

`Connection` was replaced with ` MongoClient` after `pymongo v2.8`

therefore I suggest you guys to comment in the python scripts regarding it.

TL:DR;

```
>>> import pymongo
>>> conn = pymongo.MongoClient('localhost', 27017) # not Connection anymore
>>> pymongo.version
'3.6.0'
```

because `pip` installed a recent version of pymongo for me.
You can refer [this](https://github.com/dcrosta/professor/issues/13#issuecomment-360423175) comment as well.

Peace.

**Comment by mikedory** [comment]
Hey @greed2411 - we're not longer actively maintaining this repo (as the book is also no longer being updated), but we're happy to review and discuss revisions. If you submit a pull request with this update included, we'll take a look!

**Comment by mikedory** [comment]
Awesome, thanks @greed2411! Left you a comment on the PR — and thanks again for submitting!
